2024/calcite-react/demo/src/App.tsx

```diff
@@ -29,9 +34,12 @@ function App({ webmap, title, panelHeading }: AppProps) {
       {/* Default content slot */}
       <ArcgisMap
         itemId={webmap}
+        // Step 9: Listen for on view ready
         onViewReady={(e) => {
           const { view } = e.detail;
+          // Disable popup for features widget as feature information will be rendered in side panel
           view.popupEnabled = false;
```

Comment:
should we move this into the useEffect that is in Panel.tsx as this logic better relates to that code (thus should be colocated)